src/utils/http-clients.test.js

```diff
@@ -0,0 +1,88 @@
+import { AuthenticatedHTTPClient } from './http-clients'
+import Auth0Client from '../auth/auth0-client-provider'
+import mockAxios from 'axios'
+
+describe('http-clients', () => {
+  const url = '/api/v1/dog'
+
+  describe('AuthenticatedHTTPClient', () => {
+    const token = 'token'
+    let fakeAuth0Client = {
+      isAuthenticated: (() => Promise.resolve(false)),
+      getTokenSilently: (() => Promise.resolve(token))
+    }
+    beforeEach(() => {
+      Auth0Client.get = jest.fn(() => Promise.resolve(fakeAuth0Client))
+    })
+
+    describe('.get', () => {
```

Comment:
Seeing a '.' in front end stuff makes me think 'class'

Reply:
This is best practice according to Gükenoff.